projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cds/profile-tag.e2e-spec.ts

```diff
@@ -0,0 +1,44 @@
+import { clickAllowAllFromBanner } from '../../../helpers/anonymous-consents';
+import { Navigation } from '../../../helpers/navigation';
```

Comment:
to align with other test, can you put the first letter lowercase

Reply:
I will go ahead and adjust this. Just FYI: The `clickAllowAllFromBanner` is a bit different since it is a function, whereas the Navigation is an object which has static methods.